src/core/postNote.ts

```diff
@@ -109,25 +158,67 @@ export class PostNote {
             while (images.length > 0) {
                 images[0].parentNode.removeChild(images[0]);
             }
-            this.note['body_html'] = dom.body.innerHTML;
+
+            const htmlToText = dom.body.innerHTML
+                .replace(/<\/?(table|tr|td|th)\b[^>]*>/gi, '\n\n')
+                .replace(/\r?\n/g, '\u0001')
+
+                // convert linebreak placeholders back to newlines
+                .replace(/\u0001/g, '\n');
+
+            this.note['body_html'] = htmlToText + attachmentsLinks;
+            this.note['markup_language'] = 1;
         }
     }
 
-    async addInlineAttachments(attachmentsProp: AttachmentProperties[]) {
+    async addInlineAttachments(): Promise<void> {
```

Comment:
this looks to be processing only images, not all attachments? please rename accordingly

Reply:
Inline or images, the two terms are equivalent. Inline is a term in imap references to images. `Inline indicates that the entity should be immediately displayed to the user, while attachment means that the user must take an additional action to display the entity.`

But I can change the job name if it is not clear Do you prefer to change it?

Reply:
Yes, it only works on images. Well, I renamed the function to`addInlineImages` to make it more clear.